schemas/common_definitions.json

```diff
@@ -297,5 +297,15 @@
       "additionalProperties": false,
       "required": ["goto"]
     }
+  },
+  "submission": {
+    "type": "object",
+    "properties": {
+      "button": {
```

Comment:
I realise this was the structure I suggested, but as there is no submission title property, we should consider flattening this to `submit_button`.